src/TwigHooks/src/Hookable/Renderer/HookableTemplateRenderer.php

```diff
@@ -42,13 +43,19 @@ public function render(AbstractHookable $hookable, HookableMetadata $metadata):
             return $this->twig->render($hookable->template, [
                 HooksRuntime::HOOKABLE_METADATA => $metadata,
             ]);
+        }
+        catch (HookRenderException $exception) {
+            throw $exception;
         } catch (\Throwable $exception) {
             throw new HookRenderException(
                 sprintf(
-                    'An error occurred during rendering the "%s" hook in the "%s" hookable',
+                    'An error occurred during rendering the "%s" hook in the "%s" hookable. %s',
                     $hookable->name,
                     $hookable->hookName,
+                    $exception->getMessage(),
                 ),
+                lineno: $exception->getLine(),
+                source: $exception instanceof Error ? $exception->getSourceContext() : null,
                 previous: $exception,
```

Comment:
```suggestion
                $exception->getLine(),
                $exception instanceof Error ? $exception->getSourceContext() : null,
                $exception,
```

Since the exception is only used for Hook render exception, what about having a dedicated contructor to this exception ?